nat-lab/pyproject.toml

```diff
@@ -26,11 +26,19 @@ recursive = true
 in-place = true
 
 [tool.mypy]
+check_untyped_defs = true
+enable_error_code = ['possibly-undefined', 'redundant-expr', 'unused-awaitable']
+extra_checks = true
+strict_equality = true
 exclude = [
     '^tests/protobuf$',
-    '^.*telio_bindings\\.py$',
+    '^telio_bindings\.py$',
```

Comment:
Is this correct? it does not really look like it would match anything with that `.py` at the end?

Reply:
Change from `\\.` to `\.` is correct, but `.*` should have stayed.

Reply:
Also `^tests/protobuf$` should be fixed to `^tests/protobuf`

Reply:
Fixed those 2